src/app/qgsdecorationlayoutextentdialog.cpp

```diff
@@ -42,6 +42,8 @@ QgsDecorationLayoutExtentDialog::QgsDecorationLayoutExtentDialog( QgsDecorationL
 
   updateGuiElements();
   connect( buttonBox->button( QDialogButtonBox::Apply ), &QAbstractButton::clicked, this, &QgsDecorationLayoutExtentDialog::apply );
+  connect( buttonBox, &QDialogButtonBox::helpRequested, this, &QgsDecorationLayoutExtentDialog::showHelp );
+// Pas écrit pareil ci-dessus
```

Comment:
Indeed 😄 
Btw, maybe a beginner question, what's the difference between the two ways the connect commands are written? Is any better? i tried to apply the second model to the "Apply" connection without success.